code/__HELPERS/text.dm

```diff
@@ -286,9 +286,29 @@ var/list/whitelist_name_diacritics_min = list(
 
 //Adds 'u' number of zeros ahead of the text 't'
 /proc/add_zero(t, u)
-	while (length(t) < u)
-		t = "0[t]"
-	return t
+	var/string = "[t]"
+	while (length(string) < u)
+		string = "0[string]"
+	return string
+
+//u1 = how many numbers before the dot, u2 = how many numbers after the dot
+/proc/add_zero_before_and_after(t, u1, u2)
```

Comment:
Generally I prefer a bit longer and more descriptive names like (text, beforeZeros, afterZeros). C-style short variable names suck.

Reply:
Woops, meant to make this part of a larger review.

Reply:
sure